src/client/fontengine.cpp

```diff
@@ -149,42 +153,23 @@ irr::gui::IGUIFont *FontEngine::getFont(FontSpec spec, bool may_fail)
 /******************************************************************************/
 unsigned int FontEngine::getTextHeight(const FontSpec &spec)
 {
-	irr::gui::IGUIFont *font = getFont(spec);
-
-	// use current skin font as fallback
-	if (font == NULL) {
-		font = m_env->getSkin()->getFont();
-	}
-	FATAL_ERROR_IF(font == NULL, "Could not get skin font");
```

Comment:
This and other code that assumes `font` can be null can go, `getFont()` *never* returns null